golang/src/permissions/restapi/impl/db/resource_types.go

```diff
@@ -83,7 +83,7 @@ func GetDuplicateResourceTypeByName(tx *sql.Tx, id *string, name *string) (*mode
 	defer rows.Close()
 
 	// Get the resource type.
-	var resourceTypes []*models.ResourceTypeOut
+	resourceTypes := make([]*models.ResourceTypeOut, 0)
```

Comment:
I have no doubt this is correct but for the sake of my own education, why this change?